extensions/ql-vscode/test/unit-tests/databases/db-manager.test.ts

```diff
@@ -219,6 +388,34 @@ describe("db manager", () => {
     return localDbItems[0];
   }
 
+  function getRemoteRepoDbItem(
```

Comment:
I'm note sure about this method and `getRemoteOwnerDbItem`.

It should test that the items are written to the config and can be retrieved using the `dbManager.getDbItems()` right? In that case I think it would be good to make the expectation explicit and let the current method return `repoDbItems` so that all expectations can be seen in the main test.

Reply:
For those methods, I copied the same approach as for the existing ones (`getRemoteUserDefinedListDbItem` etc). But I kind of agree that it would be nice to have the expectations more visible in the main test 🔍 

Will reshuffle those slightly! 🙏🏽

Reply:
Thanks!